js/packages/components/items/floating/FloatingItemMenuWithIconBlue.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { useThemeColor } from '@berty/store'
+
+import { ItemMenuWithIconProps } from '../interfaces'
+import { FloatingItemMenuWithIconPriv } from './FloatingItemMenuWithIcon.priv'
+
+export const FloatingItemMenuWithIconBlue: React.FC<ItemMenuWithIconProps> = props => {
```

Comment:
This variant seems overkill

Reply:
Can we test with an atomic component like that, that just do one thing ?
It's use in many files, and i just want to see if in development, this kind of component, it will be usefull or useless ?

Reply:
if you want but at least call it FloatingMenuItemWithPrimaryIcon
- FloatingItemMenu would mean it's a menu, hence the FloatingMenuItem change
- if the primary color is not blue anymore, we don't need to rename the component

Reply:
Yep the naming it's another thing, but yes i agree 👍 

FloatingMenuItemWithPrimaryIcon seems logic :)